src/npc.cpp

```diff
@@ -153,6 +154,12 @@ bool Npc::loadFromXml()
 		baseSpeed = 100;
 	}
 
+        if((attr = npcNode.attribute("pushable"))) {
+            pushable = attr.as_bool();
+        } else {
```

Comment:
Not needed. It is set to true by default in `reset()`.

Also, space after `if` and correct spacing here (use tabs instead of spaces).